src/py21cmfast/wrapper.py

```diff
@@ -944,6 +963,9 @@ def perturb_field(
             except IOError:
                 pass
 
+        # Construct FFTW wisdoms. Only if required
```

Comment:
How does the "only if required" part work? there's nothing in the Python function that stops it from executing?

Reply:
Its all encapsulated in the C function. The C function only does something **if** the ``USE_FFTW_WISDOM`` flag is passed to it, and will only create new wisdoms if it cannot locate existing ones.

I deliberately avoided logic on the Python side to minimise function complexity.

Reply:
Ah, OK cool.